src/app/Welcome/Welcome.tsx

```diff
@@ -4,25 +4,38 @@ import {
   Alert,
   AlertVariant,
   BackgroundImage,
-  Button,
+  Button, Card, CardBody, CardFooter, CardHeader, CardTitle,
   ListVariant,
   LoginPage,
   Page,
   Stack,
-  StackItem
+  StackItem, Text, TextContent, TextVariants
 } from '@patternfly/react-core';
-import back from '!!url-loader!@app/assets/images/infinispan_login_background.svg';
-import icon from '!!url-loader!@app/assets/images/infinispan_logo.svg';
+import back1 from '!!url-loader!@app/assets/images/infinispanbg_576.png';
+import back2 from '!!url-loader!@app/assets/images/infinispanbg_576@2x.png';
+import back3 from '!!url-loader!@app/assets/images/infinispanbg_768.png';
+import back4 from '!!url-loader!@app/assets/images/infinispanbg_768@2x.png';
+import back5 from '!!url-loader!@app/assets/images/infinispanbg_1200.png';
+import icon from '!!url-loader!@app/assets/images/infinispan_logo_rgb_darkbluewhite_darkblue.svg';
 import {CatalogIcon, GithubIcon, InfoIcon} from '@patternfly/react-icons'
 import {Link, useHistory} from "react-router-dom";
 import authenticationService from "../../services/authService";
 import {KeycloakService} from "../../services/keycloakService";
+import {chart_color_blue_500} from "@patternfly/react-tokens";
 
 const Welcome: React.FunctionComponent<any> = props => {
   const history = useHistory();
   const [error, setError] = useState<undefined | string>();
   const brandname = 'Infinispan';
 
+  const images = {
+    lg: back5,
+    sm: back3,
+    sm2x: back4,
+    xs: back1,
+    xs2x: back2
+  };
+
   const description1 =
     'Infinispan is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
```

Comment:
```suggestion
    '{brandname} is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. ' +
```

Reply:
Can we reduce it down to: {brandname} is a distributed in-memory key/value data store.

IMO optional schema and licensing is too much info for the welcome page.